src/utils/checkIfParamsMatches.js

```diff
@@ -0,0 +1,11 @@
+const checkIfParamsMatches = (logEntry, match) => {
```

Comment:
I suggest to rename to something like 'check that data belongs to child'

Reply:
Done